src/archivist/fetcher/fullDomFetcher.js

```diff
@@ -25,7 +24,6 @@ export default async function fetch(url, cssSelectors) {
   try {
     page = await browser.newPage();
 
-    await page.setUserAgent(new UserAgent().toString());
```

Comment:
I think this is a bad idea

See https://brightdata.com/blog/how-tos/user-agents-for-web-scraping-101

And I've read many articles that all explained that scraping should be done with a user-agent.

I believe if we remove this, we will be kicked by many servers

You'd better use it with `{"deviceCategory": "desktop"}`

Reply:
See https://github.com/ambanum/OpenTermsArchive/issues/805#issuecomment-1082793637